tasks/update-granules-metadata/tests/data/payload.json

```diff
@@ -0,0 +1,86 @@
+{
+    "config": {
+      "bucket": "cumulus-internal",
+      "buckets": {
+        "internal": {
+          "name": "cumulus-internal",
+          "type": "internal"
+        },
+        "private": {
+          "name": "cumulus-private",
+          "type": "private"
+        },
+        "protected": {
+          "name": "cumulus-protected",
+          "type": "protected"
+        },
+        "public": {
+          "name": "cumulus-public",
+          "type": "public"
+        }
+      },
+      "distribution_endpoint": "https://something.api.us-east-1.amazonaws.com/"
+    },
+    "input": {
+      "granules": [
+        {
+          "granuleId": "MOD11A1.A2017200.h19v04.006.2017201090724",
+          "files": [
+            {
+              "name": "MOD11A1.A2017200.h19v04.006.2017201090724.hdf",
+              "bucket": "cumulus-internal",
+              "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724.hdf",
+              "type": "data",
+              "fileStagingDir": "file-staging/subdir"
+            },
+            {
+              "name": "MOD11A1.A2017200.h19v04.006.2017201090724_1.jpg",
+              "bucket": "cumulus-internal",
+              "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724_1.jpg",
+              "type": "browse",
+              "fileStagingDir": "file-staging/subdir"
+            },
+            {
+              "name": "MOD11A1.A2017200.h19v04.006.2017201090724_2.jpg",
+              "bucket": "cumulus-internal",
+              "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724_2.jpg",
+              "type": "browse",
+              "fileStagingDir": "file-staging/subdir"
+            },
+            {
+              "name": "MOD11A1.A2017200.h19v04.006.2017201090724.cmr.xml",
+              "bucket": "cumulus-internal",
+              "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724.cmr.xml",
+              "type": "metadata",
+              "fileStagingDir": "file-staging/subdir"
+            }
+          ]
+        },
+        {
+          "granuleId": "MOD11A1.A2017200.h19v04.006.2017201090723",
+          "files": [
+             {
+              "name": "MOD11A1.A2017200.h19v04.006.2017201090723.cmr.json",
+              "bucket": "cumulus-internal",
+              "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090723.cmr.json",
+              "type": "metadata",
+              "fileStagingDir": "file-staging/subdir"
+            }
+          ]
+        },
+        {
```

Comment:
I'm not opposed, but what's the rationale for having three granules in the test fixture as opposed to two? Two is the sufficient minimum case to prove that multiple granules get updated.

Reply:
I see. I think you added this just for your etag tests? That's fine in that case

Reply:
Yeah, I added the third granule for my etag test